src/connectors/aws_collect.py

```diff
@@ -662,6 +662,28 @@
         ('created_at', 'TIMESTAMP_NTZ'),
         ('updated_at', 'TIMESTAMP_NTZ'),
     ],
+    # https://docs.aws.amazon.com/cli/latest/reference/ec2/describe-snapshots.html
+    'ebs_describe_snapshot': [
+        ('data_encryption_key_id', 'STRING'),
+        ('kmskeyid', 'STRING'),
+        ('recorded_at', 'TIMESTAMP_LTZ'),
```

Comment:
this has a region, so the top rows should be `recorded_at` `account_id` `region` `error`

the rest of the rows should be in the same order as in the docs